chef/cookbooks/network/recipes/default.rb

```diff
@@ -275,6 +286,10 @@ def kill_nic(nic)
       Chef::Log.info("Creating OVS bridge #{bridge} for network #{network.name}")
       Nic::OvsBridge.create(bridge)
     end
+
+    datapath_id = get_datapath_id_for_ovsbridge(bridge)
+    br.datapath_id = datapath_id unless br.datapath_id == datapath_id
```

Comment:
can this actually happen? it seems this code is only executed with network.add_ovs_bridge set, which is hopefully done only once per bridge?